shell/platform/android/build.gradle

```diff
@@ -0,0 +1,42 @@
+// This file only exists to provide IDE support to Android Studio.
+//
+// See ./README.md for details.
+
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+    dependencies {
+        classpath "com.android.tools.build:gradle:7.0.4"
+    }
+}
+
+repositories {
+    google()
+    mavenCentral()
+}
+
+apply plugin: "com.android.library"
+
+android {
+    compileSdk 31
+
+    compileOptions {
```

Comment:
I think you should remove this for durability.  There is already a gradle warning that setting this value is going to be deprecated.

Reply:
Sg. Done!